src/pages/MovieList.jsx

```diff
@@ -0,0 +1,14 @@
+import './MovieList.css'
+
+import { Cover } from '../components/Cover'
+
+
+export const MovieList=()=> {
+
+  return (
+    <div className='container'>
+      <Cover />
+      
+    </div>
+  )
+}
```

Comment:
Spacing is important to make the code clean and readable:

```
import { Cover } from '../components/Cover'

import './MovieList.css'

export const MovieList = () => {
  return (
    <div className="container">
      <Cover />
    </div>
  )
}

```

PS. Always one empty line in the end